src/aspire/basis/polar_2d.py

```diff
@@ -95,48 +95,44 @@ def evaluate(self, v):
              + v[:, half_size:, :].conj())
 
         v = m_reshape(v, (self.nrad*half_size, nimgs))
```

Comment:
Yes, oversight (its passed a vector which I think is why didn't trip any failures).

I think I've converted it now. Since I was changing the line I went ahead and replaced the nufft call (including the naturalized `many` transform) here and in evaluate_t, which I think resolves remaining cases of #171 .